### PR TITLE
Possible fix for https://github.com/wildfly/wildfly-core/pull/1086

### PR DESCRIPTION
**Pull request by kabir**
The additional commit is an attempt to fix the mixed domain failures in https://github.com/wildfly/wildfly-core/pull/1086

### CODE REVIEW

File: controller/src/main/java/org/jboss/as/controller/transform/ResourceTransformationContextImpl.java [hunk 1]
```diff
@@ -412,7 +405,7 @@ public TransformersLogger getLogger() {
         private final ImmutableManagementResourceRegistration registration;
 
         OriginalModel(Resource original, RunningMode mode, ProcessType type, TransformationTarget target, ImmutableManagementResourceRegistration registration) {
-            this.original = original;
+            this.original = original.clone();
```
Comment (bstansberry) [inline controller/src/main/java/org/jboss/as/controller/transform/ResourceTransformationContextImpl.java]: This largely defeats the point of https://github.com/wildfly/wildfly-core/pull/1102/files#diff-89ed7c94b256f549e7a6789336b54404R135 . I'm not saying it's wrong, I'm just a little sad. :(

I think I'll keep the Stage.DOMAIN operation attachment thing though. It at least saves the cost of doing a full OC.readResourceFromRoot each time, and limits the expense to just a Resource.clone().

**Comment by bstansberry** [comment]
retest this please

I want more runs to see if the OperationCancellationTestCase issue is gone.

**Comment by kabir** [comment]
Retest this please

**Comment by bstansberry** [comment]
Thanks. I've pulled this commit into my branch. I saw another failure of OperationCancellationTestCase in a run last night of my branch, so I'm not confident the problem there is fixed with my last commit. So I'm going to close this and continue with my branch with your commit in it.

**Pull request closed without merging**